docs/reference/settings/ml-settings.asciidoc

```diff
@@ -28,17 +28,19 @@ file.
 [[general-ml-settings]]
 ==== General machine learning settings
 
-`node.ml`::
-deprecated:[7.9.0,"Use <<modules-node,node.roles>> instead."]
-Set to `true` (default) to identify the node as a _machine learning node_. +
+`node.roles: [ ml ]`::
+Set `node.roles` to contain `ml` to identify the node as a _{ml} node_. +
```

Comment:
suggest append ... `to identify the node as an ML node that is capable of running jobs.`

Also, can we explain..
If using the setting `node.roles` then all required roles must be explicitly set.
The absence of `node.roles` will have ml enabled by default. However setting `node.roles:[]` will configure a dedicated coordinating node.